pkg/actions/nodegroup/scale.go

```diff
@@ -77,50 +77,64 @@ func (m *Manager) scaleUnmanagedNodeGroup(ctx context.Context, ng *api.NodeGroup
 		input.DesiredCapacity = aws.Int32(int32(*ng.DesiredCapacity))
 	}
 
-	_, err := m.ctl.AWSProvider.ASG().UpdateAutoScalingGroup(ctx, input)
-	if err != nil {
+	if _, err := m.ctl.AWSProvider.ASG().UpdateAutoScalingGroup(ctx, input); err != nil {
 		return err
 	}
-	logger.Info("nodegroup successfully scaled")
+	logger.Info("initiated scaling of nodegroup")
+
+	if wait {
+		timeoutCtx, cancel := context.WithTimeout(ctx, m.ctl.AWSProvider.WaitTimeout())
+		defer cancel()
+		if err := eks.WaitForNodes(timeoutCtx, m.clientSet, ng); err != nil {
+			return err
+		}
 
+		logger.Info("nodegroup successfully scaled")
```

Comment:
Given that we're probably not going to drain node(s) when downscaling in this PR, I'm guessing there is no effect when using the wait flag when downscaling, is this right? 🤔

Reply:
Yes, that's the behaviour right now